frontend/src/scenes/trends/trendsLogic.ts

```diff
@@ -575,6 +577,12 @@ export const trendsLogic = kea<
                 handleLifecycleDefault(cleanSearchParams, (params) => actions.setFilters(params, false))
             }
         },
+        '/insights/(:dashboardItemId)': async ({ dashboardItemId }: Record<string, string>) => {
```

Comment:
@EDsCODE What are your thoughts here too? I feel like we don't have a standardized way of calling these "dashboard items", technically it would be `/insights/dashboard_item/{id}` but we also refer to them as `panels` like here 
<img width="507" alt="Screen Shot 2021-05-05 at 10 23 30 AM" src="https://user-images.githubusercontent.com/25164963/117156675-f1940080-ad8b-11eb-972b-c6f15291c210.png">
and #3408 refers to them as `reports` 🤔

Reply:
I've decided to change it to `/insights/dashboard_item/{id}` 😄